.devops/deploy-pipelines.yml

```diff
@@ -95,6 +109,22 @@ stages:
               echo "We assume this reference to be a valid release: $(Build.SourceBranch). Therefore, there is no need to bundle a new release."
             displayName: 'Skip release bundle'
 
+
+  # Publish client SDKs to NPM
+  - stage: PublishClientSDKstoNPM
+    dependsOn: Release
+    pool:
+      vmImage: 'ubuntu-latest'
+    jobs:
+      - ${{ each spec in parameters.OPENAPI_SPEC_PATHS }}:
+        - job: publish_SDK_${{ spec.Key }}   
+          steps:
+          # Template for generating and deploying client SDk to NPM
+          - template: templates/client-sdk-publish/template.yaml@pagopaCommons
+            parameters:
+              openapiSpecPath: '${{ spec.Value }}'
+              artifactName: 'Bundle_${{ spec.Key }}'
```

Comment:
What do you think the name of each package would be?

Reply:
> What do you think the name of each package would be?

I think they should be "Bundle api_auth", "Bundle api_backend", ..